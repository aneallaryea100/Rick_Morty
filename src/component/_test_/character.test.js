import { Provider } from 'react-redux';
import TestRenderer from 'react-test-renderer';
import Character from '../character';
import store from '../../redux/configureStore';

describe('Character test', () => {
  test('snapshot', () => {
    const tree = TestRenderer.create(
      <Provider store={store}>
        <Character />
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
