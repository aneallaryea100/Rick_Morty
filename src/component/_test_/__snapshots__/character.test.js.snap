// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character test snapshot 1`] = `
<div
  className="characterMainContainer"
/>
`;
